presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -121,6 +125,53 @@ public void testPublic()
         assertThat(aliceExecutor.executeQuery(format("SELECT * FROM %s", tableName))).hasNoRows();
     }
 
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testTableOwnerPrivileges()
+    {
+        String tableName = "alice_table";
+        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
+
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+
+        onHive().executeQuery("set role admin;");
+        assertThat(onHive().executeQuery(format("SHOW GRANT USER alice ON TABLE %s", tableName)).
+                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Select only the relevant columns of SHOW GRANT
+                ImmutableList.of(
+                        row("default", tableName, "alice", "USER", "SELECT", Boolean.TRUE, "alice"),
+                        row("default", tableName, "alice", "USER", "INSERT", Boolean.TRUE, "alice"),
+                        row("default", tableName, "alice", "USER", "UPDATE", Boolean.TRUE, "alice"),
+                        row("default", tableName, "alice", "USER", "DELETE", Boolean.TRUE, "alice")
+                ));
+    }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testViewOwnerPrivileges()
+    {
+        String viewName = "alice_view";
+        String tableName = "alice_table";
+        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
+
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        onHive().executeQuery("set role admin;");
+        executeWith(createViewAs(viewName, format("SELECT * FROM %s", tableName), aliceExecutor), view -> {
```

Comment:
I don't think clean up is needed, for the same reason mentioned in https://github.com/Teradata/presto/pull/458#discussion_r93522665.